### DISCUSSION TITLE
Make translation easier for manual translation (add "save to clipboard" icon)

### DISCUSSION THREAD

**Discussion by ShaiMagal** [PrestaShop/PrestaShop#37258] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Manual e-shop translation (for example via Google Translate) is too slow, there is missing clipboard feature.

### Expected behavior

Add any icon to save text to clipboard for easier translation.

![Untitled](https://github.com/user-attachments/assets/c9aedb59-f069-4fd1-a85f-c07029a4cba1)

### Steps to reproduce

1. go to translations
2. find any translation
3. there should be any clipboard icon for copy text to clipboard, for faster translate via Google Translate

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.3

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

**Comment by kpodemski** [comment]
Hello @ShaiMagal 

This is not a bug, this is a feature request. I changed it to the discussion. Feel free to submit PR, I like the idea.
